bindings/api.go

```diff
@@ -1042,6 +1045,87 @@ func GetTorActive() bool {
 	return getBreezApp().GetTorActive()
 }
 
+func GetAddressInfo(address string) ([]byte, error) {
+	utxos, err := mempool.GetAddressUTXO(address)
+	if err != nil {
+		return marshalResponse(nil, fmt.Errorf("mempool.GetAddressUTXO, %v", err))
+	}
+	info := data.AddressInfo{
+		Address:            address,
+		ConfirmedBalance:   0,
+		UnconfirmedBalance: 0,
+		Utxos:              make([]*data.Utxo, len(utxos)),
+	}
+	for i, utxo := range utxos {
+		tx, err := mempool.GetTransactionRaw(utxo.Txid)
+		if err != nil {
+			return marshalResponse(nil, fmt.Errorf("mempool.GetTransactionRaw: %v", err))
+		}
+		if int(utxo.Vout) >= len(tx.MsgTx().TxOut) {
+			return marshalResponse(nil, fmt.Errorf("invalid wire.Tx retrieved for %s:%d", utxo.Txid, utxo.Vout))
+		}
+
+		pkScript := tx.MsgTx().TxOut[utxo.Vout].PkScript
```

Comment:
I’m going to remove this. Originally I had it written without querying mempool.space for each raw transaction to retrieve the output scripts, I decided to write it this way because I couldn’t confirm whether the output script would always be exactly the same for a given address. I have since confirmed this with a more knowledgeable developer so I will revert my change. This way we will only make two mempool.space API calls, one to get the recommended fees and one to retrieve the address UTXOs.